src/topology/continuous_function/compact.lean

```diff
@@ -427,8 +437,12 @@ begin
   simpa only [has_sum, A] using summable_of_summable_norm (hF K)
 end
 
-end weierstrass
+lemma has_sum_apply_of_locally_summable_norm {ι : Type*} {F : ι → C(X, E)}
+  (hF : ∀ K : compacts X, summable (λ i, ‖(F i).restrict K‖)) (x : X) :
+  has_sum (λ i : ι, F i x) ((∑' i : ι, F i) x) :=
+has_sum_apply (summable_of_locally_summable_norm hF).has_sum x
```

Comment:
I'm not sure if this lemma is worth having, since it follows trivially from existing lemmas and the statement is rather complex.

Reply:
Fair enough; it's only used in one or two places in my poisson-summation code, so I'll kill the separate lemma and inline it where needed.